agents/dev/stimulate_smtp_metric.py

```diff
@@ -0,0 +1,78 @@
+import logging
```

Comment:
A genuine question: do we really need to push all kinds of scripts we use in debugging to remote? Initially, `dev`'s idea was to contain a script or two that help with work that everyone does, but this is a very task-specific script.